gear/user_management/src/python/user_app/main.py

```diff
@@ -39,7 +40,8 @@ def update_email(*, proxy: FlywheelProxy, user: User, email: str) -> None:
 
 def authorize_user(*, user: User, center_id: int,
                    authorizations: Authorizations, admin_group: NACCGroup,
-                   authorization_map: AuthMap) -> None:
+                   authorization_map: AuthMap, parameter_store: ParameterStore,
```

Comment:
see other comments, want to make this unnecessary by having center group know about a `REDCapProjectRepository`